python/pyspark/sql/session.py

```diff
@@ -304,8 +304,13 @@ def __init__(
                 and not self._jvm.SparkSession.getDefaultSession().get().sparkContext().isStopped()
             ):
                 jsparkSession = self._jvm.SparkSession.getDefaultSession().get()
+                if options is not None:
+                    self._jvm.SparkSession.applyModifiableSettings(jsparkSession, options)
```

Comment:
```suggestion
                self._jvm.SparkSession.applyModifiableSettings(jsparkSession, options)
```

Can `options` be `None` now?

Reply:
should fix the type hints above ` Optional[Dict[str, Any]] =` -> `Dict[str, Any] =`

Reply:
or `Dict[str, str]`